bob-api/api/de_onderwijzers_api.php

```diff
@@ -0,0 +1,195 @@
+<?php namespace api;
+
+use ErrorException;
+
+/**
+ * Class DeOnderwijzers
+ *
+ * Wrapper for de API die de onderwijzers beschikbaar hebben gesteld.
+ *
+ * Documentatie die goed is om te lezen:
+ * - https://graphql.org/learn/queries
+ * - https://graphql.org/learn/queries/#variables
+ *
+ * Als je debug logging aan wilt hebben, zet dan $DEBUG_ENABLE naar true
+ *
+ * GraphiQl docs: http://api-test.edwh.nl/graphql/?api_key=IBFiOq8Fyoxcqxo4gyDT7pBXzC0REahqcOXu1Nql
+ *
+ * @package api
+ */
+class DeOnderwijzers
+{
+    public static $DEBUG_ENABLED = false;
+
+    /**
+     * The API key to use during the connection
```

Comment:
Deze comment is niet perse nodig

Reply:
Waarom niet? Misschien vind ik wel dat die nodig is. 

Je benoemt nu een feit, probeer het de volgende keer meer vanuit jezelf te verwoorden. Iets van: 'Mij lijkt deze comment niet perse nodig' of 'kan je mij vertellen wat deze comment toevoegt'?